specs/gloas/p2p-interface.md

```diff
@@ -135,6 +135,28 @@ def verify_data_column_sidecar(sidecar: DataColumnSidecar) -> bool:
     return True
 ```
 
+#### New `is_higher_value_bid`
+
+```python
+def is_higher_value_bid(
+    current_bid: ExecutionPayloadBid,
+    new_bid: ExecutionPayloadBid,
+) -> bool:
+    """
+    Return ``True`` if ``new_bid`` is at least ``MIN_BID_INCREASE_PERCENT``
+    percent more profitable than ``current_bid``.
+    """
+    if new_bid.value <= current_bid.value:
+        return False
+
+    quotient = current_bid.value // 100
+    remainder = current_bid.value % 100
+    min_increase = quotient * MIN_BID_INCREASE_PERCENT
+    min_increase += (remainder * MIN_BID_INCREASE_PERCENT + 99) // 100
```

Comment:
This is a round up. I'm not following the full discussion on this, but do people think percentage gives us enough precision?